assets/js/script.js

```
@@ -0,0 +1,231 @@
+        });
+    });
+
+    // open chat box
+    $('.displayChatbox').on('click', function() {
+        var my_val = $('.pcoded').attr('vertical-placement');
+        if (my_val == 'right') {
+            var options = {
+                direction: 'left'
+            };
+        } else {
+            var options = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'options'. Consider to delete it or rename it.

'options' is defined here:


(lint/suspicious/noRedeclare)